pillar/base/haproxy.sls

```diff
@@ -75,7 +75,7 @@ haproxy:
         - {{ config.server_name }}
       verify_host: bugs.psf.io
       check: "HEAD / HTTP/1.1\\r\\nHost:\\ {{ config.server_name }}"
-      rate_limit: {{ config.get('rate_limit', 10) }}
+      rate_limit: {{ config.get('rate_limit', 100) }}
```

Comment:
I’m nearly certain roundup can’t handle 100 rpm, and I _think_ we’ll see effectively 2x the rate since the counters aren’t shared between haproxy instances. So it’s not clear if this will result in any significant defense.